README.md

```diff
@@ -7,10 +7,10 @@ projects using [Packer](https://www.packer.io/).
 
 ## Available Hooks ##
 
-| Hook name         | Description                                             |
-| ----------------- | ------------------------------------------------------- |
-| `packer_validate` | Validate all Packer templates.                          |
-| `packer_fmt`      | Check that Packer HCL templates are properly formatted. |
+| Hook name         | Description                                                                           |
+| ----------------- |---------------------------------------------------------------------------------------|
```

Comment:
I'm not sure why these spaces were removed, but we should be consistent with our Markdown formatting here.
```suggestion
| ----------------- | ------------------------------------------------------------------------------------- |
```